internal/k8s/kind.go

```diff
@@ -73,8 +70,6 @@ func VersionOf(obj interface{}) string {
 		switch obj := obj.(type) {
 		case *v1.Secret, *v1.Service, *v1.Endpoints:
 			return v1.SchemeGroupVersion.String()
-		case *v1beta1.Ingress:
```

Comment:
this function doesn't have a case for Ingress v1, I assume it should (haven't looked at where it's used)

Reply:
ah good call, missed that one 👍🏽

Reply:
seems a good few of these cases aren't actually hit with the tests, the coverage report shows

i don't know the history on this code but maybe this could be shortened 🤔

Reply:
Yeah, I don't really know either. Looks like the first line in the function might be the one that's actually hit to get the `Kind` for these objects. In any case, update looks good.